module_4/src/com/epam/aggregationandcomposition/start/Controller.java

```diff
@@ -0,0 +1,73 @@
+package com.epam.aggregationandcomposition.start;
+
+import com.epam.aggregationandcomposition.entity.lexemes.Readable;
+import com.epam.aggregationandcomposition.entity.lexemes.*;
+import com.epam.aggregationandcomposition.exceptions.LexemeLogicalException;
+
+import java.util.ArrayList;
+
+public class Controller {
+
+    private static final String[] WORDS_PART1 = {"Java", "is", "a", "set", "of", "computer", "software", "and",
+            "specifications", "developed", "by", "James", "Gosling", "at", "Sun", "Microsystems"};
+    private static final String[] WORDS_PART2 = {"The", "Oracle", "Corporation", "provides", "a", "system", "for",
+            "developing", "application", "software"};
+    private static final String[] WORDS_PART3 = {"deploying", "it", "in", "a", "cross-platform", "computing",
+            "environment"};
+    private static final String[] WORDS_PART4 = {"Java", "is", "used", "in", "a", "wide", "variety", "of",
+            "computing", "platforms"};
+    private static final String[] PUNCTUATION = {".", ","};
+    private static final String ADDITIONAL_TEXT = "Java is the best!";
+
+    public static void main(String[] args) {
+
+        // --- TASK 01 ---
+        try {
+            Readable dot = new Punctuation(PUNCTUATION[0]);
+            Readable comma = new Punctuation(PUNCTUATION[1]);
+
+            ArrayList<Readable> lexemesForSentence1 = new ArrayList<>();
```

Comment:
Make lexemesForSentence1 List<Readable>
```suggestion
            List<Readable> lexemesForSentence1 = new ArrayList<>();
```